openedx_webhooks/lib/jira/test/test_lib_jira_models.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from openedx_webhooks.lib.exceptions import NotFoundError
+
+
+class TestJiraFields:
+    def test_get_by_name(self, fields):
+        field = fields.get_by_name('test01')
+        assert field.name == 'test01'
+        assert field.first is True
+
+    def test_get_by_name_no_result(self, fields):
+        with pytest.raises(NotFoundError):
+            fields.get_by_name('random')
+
+
+class TestJiraField:
+    def test_getattr(self, field):
+        assert field.name == 'test01'
+        assert field.first is True
+
+    def test_getattr_error(self, field):
+        with pytest.raises(AttributeError):
+            field.nope
```

Comment:
That we fulfill the contract of the method and raise an appropriate error when the `JiraField` object doesn't contain an attribute.